src/Grpc.Core.Api/README.md

```diff
@@ -0,0 +1,3 @@
+# Grpc.Core.Api
+
+`Grpc.Core.Api` is the shared API package for gRPC C# and gRPC for .NET implementations of gRPC.
```

Comment:
Packages in grpc-dotnet have README.md embedded in nupkg to display on nuget. Should add an issue with the list of new projects without embedded readme.

Reply:
the .csproj already has `<PackageReadmeFile>README.md</PackageReadmeFile>` isn't that sufficient?

Reply:
Also need https://github.com/grpc/grpc-dotnet/blob/f238855fcb77cec01e87eb9cc08680c3420c132f/src/Grpc.AspNetCore/Grpc.AspNetCore.csproj#L17

Reply:
done.